integration-tests/entry-point.sh

```diff
@@ -60,6 +57,16 @@ chmod 640 ${CONFIG_PATH}
 # start ssh
 service ssh start || true
 
+# Start Device Client for a short time without lockfile dir
+rm -rf /run/lock
+./aws-iot-device-client 2>&1 &
+sleep 2
```

Comment:
This actually makes the code below which waits for the runtime config file useless.

Also, /run/lock doesn't exist already so it isn't really testing anything.

Maybe create the lock file, verify DC fails to start, then delete it?

Reply:
i see, creating the lockfile will be tricky without starting the device client first since it needs a valid PID to be written into it

Reply:
I think instead of sleeping here you should be verifying that the DC exited?